snorkel/mtl/snorkel_config.py

```diff
@@ -0,0 +1,50 @@
+default_config = {
```

Comment:
Sooooo why isn't this a JSON?

Reply:
So that it can be directly imported easily instead of having to use a json reader each time, and so there's no question about how types are being converted.